executor/adapter.go

```diff
@@ -324,6 +324,9 @@ func (a *ExecStmt) buildExecutor(ctx sessionctx.Context) (Executor, error) {
 	return e, nil
 }
 
+// QueryReplacer replaces new line and tab for grep result including query string.
+var QueryReplacer = strings.NewReplacer("\r", "", "\n", " ", "\t", " ")
```

Comment:
maybe use `var QueryReplacer = strings.NewReplacer("\r", " ", "\n", " ", "\t", " ")`?
if len(new) == len(old) == 1 replace will be more quicker~

```
func BenchmarkPureBinary(b *testing.B) {
	var r = strings.NewReplacer(",", " ", "-", " ")
	for i := 0; i < b.N; i++ {
		a := r.Replace("sadfsadfa,sdfsd,dsfsa,sdfsad,dsfsa-sdfa-sdf-dsfad-fsadf-sa,sadfas")
		a = a
	}
}

func BenchmarkBinaryString(b *testing.B) {
	var r = strings.NewReplacer(",", "", "-", "")
	for i := 0; i < b.N; i++ {
		a := r.Replace("sadfsadfa,sdfsd,dsfsa,sdfsad,dsfsa-sdfa-sdf-dsfad-fsadf-sa,sadfas")
		a = a
	}
}

BenchmarkPureBinary-8     	10000000	       144 ns/op	     160 B/op	       2 allocs/op
BenchmarkBinaryString-8   	 5000000	       267 ns/op	     128 B/op	       2 allocs/op
```